ecs/wait.go

```diff
@@ -28,8 +28,12 @@ import (
 	"github.com/aws/aws-sdk-go/aws"
 )
 
-func (b *ecsAPIService) WaitStackCompletion(ctx context.Context, name string, operation int) error { //nolint:gocyclo
+func (b *ecsAPIService) WaitStackCompletion(ctx context.Context, name string, operation int, ignored ...string) error { //nolint:gocyclo
 	knownEvents := map[string]struct{}{}
+	for _, id := range ignored {
```

Comment:
Looking at the code, was this causing issues ? at first sight it seems it would refresh the progress display for these "ignored" events but should redisplay the same things. I guess this was not the case, to require this additional `ignored` params

Reply:
issue is we show all resources as "CREATE_COMPLETE" and this override the "DELETE_" status that we just created for capacity provider. Always has been weird to me we first display this "CREATE" or "UPDATE" status for previous stack operation.